src/defi-cli.cpp

```diff
@@ -60,6 +60,9 @@ static void SetupCliArgs()
     gArgs.AddArg("-rpcwallet=<walletname>", "Send RPC for non-default wallet on RPC server (needs to exactly match corresponding -wallet option passed to defid). This changes the RPC endpoint used, e.g. http://127.0.0.1:8554/wallet/<walletname>", ArgsManager::ALLOW_ANY, OptionsCategory::OPTIONS);
     gArgs.AddArg("-stdin", "Read extra arguments from standard input, one per line until EOF/Ctrl-D (recommended for sensitive information such as passphrases). When combined with -stdinrpcpass, the first line from standard input is used for the RPC password.", ArgsManager::ALLOW_ANY, OptionsCategory::OPTIONS);
     gArgs.AddArg("-stdinrpcpass", "Read RPC password from standard input as a single line. When combined with -stdin, the first line from standard input is used for the RPC password.", ArgsManager::ALLOW_ANY, OptionsCategory::OPTIONS);
+    gArgs.AddArg("-walletfastselect", strprintf("Whether to skip IsSolvable signable UTXO check and return on first valid auth. (default: %u)", DEFAULT_FAST_SELECT), ArgsManager::ALLOW_ANY, OptionsCategory::OPTIONS);
+    gArgs.AddArg("-walletselectionfastsolvable", strprintf("Whether to skip IsSolvable signable UTXO check. (default: %u)", DEFAULT_FAST_SOLVABLE), ArgsManager::ALLOW_ANY, OptionsCategory::OPTIONS);
+    gArgs.AddArg("-walletselectionexitonfirstmatch", strprintf("Whether to return on first valid auth. (default: %u)", DEFAULT_EXIT_ON_FIRST_MATCH), ArgsManager::ALLOW_ANY, OptionsCategory::OPTIONS);
```

Comment:
I imagined something like this while suggesting these:

- `wallet-fast-select`
- `wallet-selection-skip-solver`
- `wallet-selection-exit-on-first-match`

But I realize our conention is single word.. Hmm

- `walletfastselect`
- `walletselectoptskipsolver`
- `walletselectoptearlyexit`

Perhaps?